cl/phase1/network/services/block_service.go

```diff
@@ -169,7 +169,7 @@ func (b *blockService) publishBlockEvent(block *cltypes.SignedBeaconBlock) {
 
 // scheduleBlockForLaterProcessing schedules a block for later processing
 func (b *blockService) scheduleBlockForLaterProcessing(block *cltypes.SignedBeaconBlock) {
-	log.Debug("Block scheduled for later processing", "block", block.Block.Slot)
+	log.Debug("Block scheduled for later processing", "slot", block.Block.Slot, "block", block.Block.Body.ExecutionPayload.BlockNumber)
```

Comment:
Possible nil panic?

Reply:
No, the block was generated by this code and the ExecutionPayload was initialized:
https://github.com/ledgerwatch/erigon/blob/78bd2bc7c48d71c7c387b20c1f4aa9ec46409875/cl/phase1/network/gossip_manager.go#L162-L168
https://github.com/ledgerwatch/erigon/blob/78bd2bc7c48d71c7c387b20c1f4aa9ec46409875/cl/cltypes/beacon_block.go#L97-L99
https://github.com/ledgerwatch/erigon/blob/78bd2bc7c48d71c7c387b20c1f4aa9ec46409875/cl/cltypes/beacon_block.go#L129-L131
https://github.com/ledgerwatch/erigon/blob/78bd2bc7c48d71c7c387b20c1f4aa9ec46409875/cl/cltypes/beacon_block.go#L147-L160
https://github.com/ledgerwatch/erigon/blob/78bd2bc7c48d71c7c387b20c1f4aa9ec46409875/cl/cltypes/eth1_block.go#L63-L65